scripts/schema.sql

```diff
@@ -130,4 +130,10 @@ create table if not exists header_chain_proofs (
     proof bytea
 );
 
+-- Verifier table of Winternitz public keys for every operator and time_tx pair
+create table if not exists winternitz_public_keys (
+    watchtower_id int primary key not null,
```

Comment:
Let's save winternitz per operator_id not watchtower_id so that we don't have to load all winternitz pubkeys when signing